src/components/ControlPanel/controlPanelSections.js

```diff
@@ -0,0 +1,16 @@
+import ControlPanelIconImg from './account_icon.png';
+
+export const controlPanelSections = [
+  {
```

Comment:
No estoy seguro de si es buena idea hacerlo ahora, siempre se puede refactorizar después.

Pero me parece que a la larga, esto no va a ser una constante, ya que los items que se muestran y algunos detalles van a depender del usuario. ¿Que opinás de darle la forma de un servicio?

```suggestion
export const getControlPanelSections() => [
  {
```

Incluso para esto se podría usar el composition root y permitir inyectarlo, pero ahí si me parece que es un cambio más grande y que tal vez no sea útil.

Reply:
> https://github.com/FromDoppler/doppler-webapp/pull/1662#discussion_r728986670

Cual seria la diferencia con lo actual? Imagino que tendra algun beneficio pero no se cual es 😛

Reply:
Joya, la alternativa 3 era la que no entendia. Lo tengo presente para cuando lo modifique. Gracias!